### PR TITLE
chore(ci): add possibility to auto-update PRs

### PR DESCRIPTION
**Pull request by ableuler**
This PR adds the sibling of `automerge`. Set both the `autoupdate` and `automerge` label on an accepted PR and let the 🤖 do the rest.

### CODE REVIEW

**Comment by rokroskar** [comment]
Why not add the auto-update label on the auto-PRs?

**Comment by ableuler** [comment]
> Why not add the auto-update label on the auto-PRs?

Yeah, good point. Let's merge and see if it does what we want on a PR where we manually add this label. If it behaves as expected we should definitely add this automatically to the auto-update PRs.

**Comment by rokroskar** [comment]
I don't think it does because it should have auto updated this pr already

File: .github/workflows/autoupdate.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  # This will trigger on all pushes to all branches.
+  push: {}
+jobs:
+  autoupdate:
+    name: autoupdate
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action:v1
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (rokroskar) [inline .github/workflows/autoupdate.yaml:12]: should use the renkubot token here

**Review by rokroskar — CHANGES_REQUESTED** [review]

File: .github/workflows/autoupdate.yaml (hunk 1 above)
Comment (ableuler) [reply .github/workflows/autoupdate.yaml]: thx for pointing this out!

**Review by rokroskar — APPROVED** [review]

**Pull request merged**